pkg/preflights/host-preflight.yaml

```diff
@@ -370,6 +424,342 @@ spec:
+              message: Port 9091/TCP is available
+          - error:
+              message: Port 9091/TCP is required, but an unexpected error occurred when trying to connect to it
+    - tcpPortStatus:
+        checkName: Kube API Server Port Availability
+        collectorName: Kube API Server Port
+        outcomes:
+          - fail:
+              when: "connection-refused"
+              message: Port 6443/TCP is required, but the connection was refused. This may indicate that the connection is being redirected to somewhere else on your network (perhaps a proxy configuration).
+          - fail:
+              when: "address-in-use"
+              message: Port 6443/TCP is required, but another process is already listening on it. Relocate the conflicting process to continue.
+          - fail:
+              when: "connection-timeout"
+              message: Port 6443/TCP is required, but the connection timed out. Ensure that your firewall doesn't block port 6443/TCP.
+          - fail:
+              when: "error"
+              message: Port 6443/TCP is required, but an unexpected error occurred when trying to connect to it
+          - pass:
+              when: "connected"
+              message: Port 6443/TCP is available
+          - error:
+              message: Port 6443/TCP is required, but an unexpected error occurred when trying to connect to it
+    - tcpPortStatus:
+        checkName: Envoy Port Availability
+        collectorName: Envoy Port
+        outcomes:
+          - fail:
+              when: "connection-refused"
+              message: Port 7443/TCP is required, but the connection was refused. This may indicate that the connection is being redirected to somewhere else on your network (perhaps a proxy configuration).
+          - fail:
+              when: "address-in-use"
+              message: Port 7443/TCP is required, but another process is already listening on it. Relocate the conflicting process to continue.
+          - fail:
+              when: "connection-timeout"
+              message: Port 7443/TCP is required, but the connection timed out. Ensure that your firewall doesn't block port 7443/TCP.
+          - fail:
+              when: "error"
+              message: Port 7443/TCP is required, but an unexpected error occurred when trying to connect to it
+          - pass:
+              when: "connected"
+              message: Port 7443/TCP is available
+          - error:
+              message: Port 7443/TCP is required, but an unexpected error occurred when trying to connect to it
+    - tcpPortStatus:
+        checkName: Kotsadm Node Port Availability
+        collectorName: Kotsadm Node Port
+        outcomes:
+          - fail:
+              when: "connection-refused"
+              message: Port 30000/TCP is required, but the connection was refused. This may indicate that the connection is being redirected to somewhere else on your network (perhaps a proxy configuration).
```

Comment:
I need to better understand this one to comment on the wording, but this can be fine for now.

Reply:
For context: this means that we have been able to start listening on a given port (the port is available) but when we tried to connect to it we failed. This is an IMHO unlikely scenario.

We have discussed this personally and we came to the conclusion that this could be caused by an erroneous proxy configuration but I am not sure this is even possible as we are testing at TCP level, not HTTP.  Some iptables rules may cause this though (redirecting traffic somewhere else) but again, unlikely.